src/extra_checks/checks/model_field_checks.py

```diff
@@ -305,8 +305,10 @@ def apply(
         choices = field.flatchoices  # type: ignore
         if choices:
             field_choices = [c[0] for c in choices]
-            if field.blank and "" not in field_choices:
+            if field.blank and "" not in field_choices and field.empty_strings_allowed:
                 field_choices.append("")
+            if field.null and None not in field_choices:
+                field_choices.append(None)
```

Comment:
Thanks for PR, I believe we don't need to add null as a choice to constraint, it should work if column is nullable.

Reply:
Okay, I'll check that and remove the code :)

Reply:
Removed the None check as it is not necessary (tested with Postgres)